packages/theme/src/theme/styles/math.css

```
@@ -0,0 +1,31 @@
+.vp-doc mjx-container[display='true'] {
+
+  display: block;
```

Stylelint (17.4.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)